src/lib.rs

```diff
@@ -150,10 +190,17 @@
 #![warn(missing_docs)]
 #![cfg_attr(feature = "nightly", feature(panic_handler))]
 
+#![cfg_attr(all(not(feature = "use_std"), not(test)), no_std)]
```

Comment:
I tried it and it doesn't work because of the `extern crate core as std;` line just below. I think it's fine to just leave the this line as it is.